release/post-release.sh

```diff
@@ -50,26 +52,26 @@ check_operators() {
   do
     echo "Operator: $OPERATOR"
     if [ ! -d "$TEMP_RELEASE_FOLDER/$OPERATOR" ]; then
-      echo "Expected folder is missing: $TEMP_RELEASE_FOLDER/$OPERATOR"
-      exit 1
+      echo "Cloning folder: $TEMP_RELEASE_FOLDER/$OPERATOR"
+      # $TEMP_RELEASE_FOLDER has already been created in main()
+      git clone "git@github.com:stackabletech/${OPERATOR}.git" "$TEMP_RELEASE_FOLDER/$OPERATOR"
     fi
-
     cd "$TEMP_RELEASE_FOLDER/$OPERATOR"
 
     DIRTY_WORKING_COPY=$(git status --short)
```

Comment:
I'm not sure this catches things that might have been staged. I have another command that checks for this:

https://github.com/stackabletech/demos/blob/de005daa945d86337edfe1c9e9e848b30208dd2e/.scripts/update_refs.sh#L21-L25

Reply:
Updated: 438a466